test/functional/api/services_controller_test.rb

```diff
@@ -0,0 +1,106 @@
+require 'test_helper'
+
+class Api::ServicesControllerTest < ActionController::TestCase
+
+  def setup
+    @provider  = FactoryBot.create(:provider_account)
+    @service   = FactoryBot.create(:service, account: @provider)
+    member     = FactoryBot.create(:member, account: @provider)
+    permission = FactoryBot.create(:member_permission, admin_section: :plans)
+
+    member.member_permissions << permission
+
+    @request.host = @provider.domain
+
+    login_as member
+  end
+
+  def test_load_and_authorize_resource
+    get :show, id: FactoryBot.create(:service).id
```

Comment:
these functional tests are deprecated... plz use integration tests instead

Reply:
I don't know how this ended up here..